pkg/catalog/outbound_traffic_policies.go

```diff
@@ -75,9 +76,10 @@ func (mc *MeshCatalog) listOutboundTrafficPoliciesForTrafficSplits(sourceNamespa
 
 		var weightedClusters []service.WeightedCluster
 		for _, backend := range split.Spec.Backends {
-			ms := service.MeshService{Name: backend.Service, Namespace: split.ObjectMeta.Namespace}
+			//ms := service.MeshService{Name: backend.Service, Namespace: split.ObjectMeta.Namespace, ClusterName: "local"}
 			wc := service.WeightedCluster{
-				ClusterName: service.ClusterName(ms.String()),
```

Comment:
This is different.. this is an envoy cluster, and needs to be set from the meshservice.